src/styles/content.scss

```diff
@@ -0,0 +1,126 @@
+.card {
+  display: grid;
+  justify-content: center;
+  padding: 0 88px;
+
+  &__container {
+    display: grid;
+    row-gap: 46px;
+    column-gap: 48px;
+    grid-template-columns: repeat(4, $card-width);
+
+    padding-top: 50px;
+    padding-bottom: 72px;
+    justify-content: center;
+    margin-left: 48px;
+
+    @media (min-width: 488px) {
+      grid-template-columns: repeat (2, $card-width);
+    }
+
+    @media (min-width: 768px) {
+      grid-template-columns: repeat (3, $card-width);
+    }
+
+    @media (min-width: 1024px) {
+      grid-template-columns: repeat (4, $card-width);
+      margin: 0 40px;
+    }
+  }
+
+  &__item {
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 0;
+    width: $card-width;
+    transition: transform 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    margin: 0 3px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)